### PR TITLE
Blog Hawkular APM OpenTracing Javascript

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
+= Hawkular APM OpenTracing JavaScript
+Pavol Loffay
+2016-11-16
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, APM, microservices, distributed tracing, javascript
+
+In the latest https://github.com/hawkular/hawkular-apm/releases/tag/0.12.0.Final[release]
+of Hawkular APM we have added new OpenTracing implementation. This
```
Comment (objectiser) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: added "a" new

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+time we decided to add JavaScript provider, therefore, now it is possible to get timing metrics
+from polyglot environment instrumented with our Java and JavaScript OpenTracing implementations.
```
Comment (objectiser) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: from "a" polyglot

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+== Library
+Library is implemented against the latest (0.13.0) version of OpenTracing JavaScript API.
+Data is reported to APM via HTTP. For debugging it is possible to use a console recorder.
+From the sampling perspective it allows developers to add their own sampling mechanism.
+In the future releases we will add more built-in samplers.
+
+Following code snippet show how to initialize and create spans in JavaScript code. It works for
```
Comment (objectiser) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: "The following code snippet shows"

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+const span = opentracing.globalTracer().startSpan('name');
+span.finish();
+```
+
+After the initialization tracer instance can be obtained via `opentracing.globalTracer()` method.
+
+== Want To Try It?
+Our https://github.com/hawkular/hawkular-apm-opentracing-javascript/tree/master/examples[Github] repository contains
+two example applications. Ono if which is tracing requests from a web browser across Node.js
```
Comment (objectiser) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: "One of which"

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+backed services. Examples contains detailed instructions how to run them.
```
Comment (objectiser) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: 'contain' and 'instructions on how'

**Review by objectiser — APPROVED** [review]
Looks good - just some typos/grammer suggestions.

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc (hunk 4 above)
Comment (pavolloffay) [reply src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: Uno of which :smile:

**Comment by pavolloffay** [comment]
@objectiser  thanks!

@pilhuhn @theute Could you please look at this too?

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+Pavol Loffay
+2016-11-16
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, APM, microservices, distributed tracing, javascript
+
+In the latest https://github.com/hawkular/hawkular-apm/releases/tag/0.12.0.Final[release]
+of Hawkular APM we have added a new OpenTracing implementation. This
+time we decided to add JavaScript provider, therefore, now it is possible to get timing metrics
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: add a JavaScript ?

now it is -> it is now (?)

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+from a polyglot environment instrumented with our Java and JavaScript OpenTracing implementations.
+
+== Library
+Library is implemented against the latest (0.13.0) version of OpenTracing JavaScript API.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: The library ..

latest (0.13 .) -> latest version (0.13) ?

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+Data is reported to APM via HTTP. For debugging it is possible to use a console recorder.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: For debugging purposes

File: src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+From the sampling perspective it allows developers to add their own sampling mechanism.
+In the future releases we will add more built-in samplers.
+
+The following code snippet shows how to initialize and create spans in JavaScript code. It works for
+a web browser and Node.js application.
+
+``` javascript
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc]: Adoc uses ---- to surround code (and yes I see that it is working like you did it) and you can use  

```
.Title of the snipped
[source, javascript]
----
my code
---- 

```

**Comment by pavolloffay** [comment]
@pilhuhn thanks!

**Pull request merged**